lti_consumer/lti_1p1/consumer.py

```diff
@@ -241,6 +244,14 @@ def set_custom_parameters(self, custom_parameters):
 
         self.lti_custom_parameters = custom_parameters
 
+    def set_extra_claims(self, claim):
+        """
+        Sets launch extra claims
+        """
+        if not isinstance(claim, dict):
+            raise ValueError('Invalid extra claim: is not a dict.')
```

Comment:
It's helpful to provide data about the bad value.  Perhaps:
```
raise ValueError('Invalid extra claim: {!r} is not a dict.'.format(claim))
```

Reply:
Sounds good. Added :+1: